README.md

```diff
@@ -71,6 +71,7 @@ $ venv
 Install CommCare Export via `pip`
 
 ```shell
+$ pip install setuptools
```

Comment:
nit: Should we make a comment that this is for version 3.12?

Reply:
I just tested this in Python 3.12. Turns out we don't need setuptools if we install using `pip install commcare-export`. I'll drop this line.

Reply:
3974ae6